<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>637</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>789</width>
    <height>637</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>789</width>
    <height>637</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>789</width>
     <height>637</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>789</width>
     <height>637</height>
    </size>
   </property>
   <widget class="QWidget" name="mainmenu_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>721</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>881</width>
      <height>721</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>881</width>
      <height>721</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 255);</string>
    </property>
    <widget class="QLabel" name="mainmenu_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>301</width>
       <height>54</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25pt &quot;Berlin Sans FB&quot;;

color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Main Menu</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="playbutton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>380</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;
font: 34pt &quot;Berlin Sans FB&quot;;
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QLabel" name="username_label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25pt &quot;Berlin Sans FB&quot;;

color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="quitbutton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>380</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:20px;font: 34pt &quot;Berlin Sans FB&quot;;
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)</string>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progress_progressBar">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="level_label">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>290</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25pt &quot;Berlin Sans FB&quot;;
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)</string>
     </property>
     <property name="text">
      <string>Level :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
